core/server/models/base/index.js

```diff
@@ -79,25 +79,41 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
             this.include = _.clone(options.include);
         }
 
-        this.on('creating', this.creating, this);
-        this.on('saving', function onSaving(model, attributes, options) {
-            return Promise.resolve(self.saving(model, attributes, options)).then(function then() {
-                return self.validate(model, attributes, options);
+        ['creating', 'created', 'updated', 'destroying', 'destroyed', 'saved']
```

Comment:
Is `updating` deliberately missing? Just checking - I don't think we use it, but it would seem like a good idea to not miss any events?